pyproject.toml

```diff
@@ -106,5 +106,5 @@ deps =
   coverage[toml]>=5.3
 commands =
     codecov --required
-depends = py{,38,34,35,36,37,py3}-test-{twlatest,twtrunk,dev}
+depends = py{,35,36,37,38,39,py3}-test-{twlatest,twtrunk,dev}
```

Comment:
is there any semantic for the leading comma ? Just tasking :)

Reply:
ah this is for native python eg `py-test-twlatest`